cfme/configure/configuration/region_settings.py

```diff
@@ -981,8 +981,7 @@ class ReplicationGlobalAddView(ReplicationView):
     username = Input(name='userid')
     password = Input(name='password')
     accept_button = Button('Accept')
-    action_dropdown = Dropdown(
-        "//*[@id='form_div']//table//button[contains(@class, 'dropdown-toggle')]")
+    action_dropdown = Dropdown("")
 
```

Comment:
Is it possible to find better xpath for this widget?

Reply:
@ganeshhubale  please see in the comment.